src/components/WalletConnect.vue

```diff
@@ -4,43 +4,43 @@
 			<a-col>
 				<a-dropdown>
 					<template #overlay>
-						<a-menu @click="handleNetworkConnect">
-							<a-menu-item :key="NetworkTypes.MAIN_NET"> MainNet </a-menu-item>
-							<a-menu-item :key="NetworkTypes.TEST_NET"> TestNet </a-menu-item>
-							<a-menu-item :key="NetworkTypes.BETA_NET"> BetaNet </a-menu-item>
-							<!-- only for testing -->
-							<a-menu-item :key="NetworkTypes.PRIVATE_NET">
-								PrivateNet
+						<a-menu @click="handleWalletConnect">
+							<a-menu-item :key="WalletType.ALGOSIGNER">
+								Algosigner
+							</a-menu-item>
+							<a-menu-item :key="WalletType.MY_ALGO">
+								MyAlgo Wallet
+							</a-menu-item>
+							<a-menu-item :key="WalletType.WALLET_CONNECT">
+								Wallet Connect
 							</a-menu-item>
 						</a-menu>
 					</template>
 					<a-button>
-						<span v-if="selectedNetwork === NetworkTypes.NONE">
-							Select Network</span
-						>
-						<span v-else>Selected Network : {{ selectedNetwork }}</span>
+						Connect Wallet
 						<DownOutlined />
 					</a-button>
 				</a-dropdown>
 			</a-col>
 
-			<a-col v-if="selectedNetwork !== NetworkTypes.NONE">
+			<a-col v-if="selectedWallet !== 0">
```

Comment:
Could we use networkTypes.NONE instead of "0" at here?

Reply:
oh yes, updated